src/scenes/Summary/summaryTable.ts

```diff
@@ -116,6 +116,13 @@ function getSummaryTableQueryRunner(metrics: DataSourceRef, sm: DataSourceRef) {
   const transformed = new SceneDataTransformer({
     $data: queryRunner,
     transformations: [
+      {
+        id: 'renameByRegex',
+        options: {
+          regex: 'target',
+          renamePattern: 'instance',
+        },
+      },
```

Comment:
This is the only actual change, as @peterschretlen pointed out in Slack:
The checks data has the `id` we need for the path. But since it doesn't have a value for `instance`, (the field is instead called `target`) then it's not in the final results after going through all those transformations https://github.com/grafana/synthetic-monitoring-app/blob/main/src/scenes/Summary/summaryTable.ts

So, adding this transformation at the top will fix it. Otherwise, by the time the data reaches that last transformation, `target` is no longer there (it gets lost after the join)